openwisp_radius/tests/test_utils.py

```diff
@@ -23,7 +23,13 @@ def test_validate_file_format(self):
             'Unrecognized file format, the supplied file does not look like a CSV file.'
             in error.exception.message
         )
-
+        
+        utf_16_file_1_format_path = self._get_path('static/test_batch_utf16_file1.csv')
```

Comment:
These assertions deserve to be in a dedicated test

Reply:
done.